contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -255,9 +255,10 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
       return tokenPrice;
     }
 
-    // If the token price feed is set, return the price from the feed
-    // The price feed is the fallback because we do not expect it to be the default source due to the gas cost of reading from it
-    return _getTokenPriceFromDataFeed(priceFeedConfig);
+    // If the token price feed is set, retrieve the price from the feed
+    Internal.TimestampedPackedUint224 memory oraclePrice = _getTokenPriceFromDataFeed(priceFeedConfig);
```

Comment:
nit: rename to `feedPrice` to be consistent with the rest of the codebase?

Reply:
https://github.com/smartcontractkit/chainlink/pull/14854/commits/7a905503b48bd4c352cc58d35f0016a349aba301